pl_examples/domain_templates/computer_vision_fine_tuning.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, milestones: tuple = (5, 10), train_bn: bool = False):
         self.train_bn = train_bn
 
     def freeze_before_training(self, pl_module: pl.LightningModule):
-        self.freeze(modules=pl_module.feature_extractor, train_bn=self.train_bn)
+        self.freeze(modules=pl_module.feature_extractor, train_bn=False)
```

Comment:
Why this change? Is it related to the fix?

Reply:
Just to make training faster, but still slow.

Reply:
train_bn is set to False by default, but the argument wasn't correctly passed. Fixed in https://github.com/PyTorchLightning/pytorch-lightning/pull/6027/commits/efcff122e7c8560c1699dbb82426a433f7a75527